tests/cases/fourslash/javascriptModules22.ts

```diff
@@ -29,4 +29,4 @@ verify.completionListContains("name");
 edit.insert("name;\nsausages.");
 verify.completionListContains("eggs");
 edit.insert("eggs;");
-verify.noErrors();
+verify.noErrors({ ignoreInfoDiagnostics: true });
```

Comment:
Personally, I would expect `noDiagnostics` to mean "no diagnostics of any severity" and `noErrors` to mean "no diagnostics of error severity".